app/Appointments.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Appointments extends Model
+{
+    protected $fillable = [
+        'client_id', 'date_time','type','price',
+    ];
+}
```

Comment:
Can we add a trailing line at the end of the class: https://stackoverflow.com/questions/9921927/php-is-there-a-reason-to-put-a-newline-at-the-end-of-file